go.mod

```diff
@@ -74,6 +74,7 @@ require (
 	github.com/googleapis/gax-go/v2 v2.12.0 // indirect
 	github.com/hashicorp/hcl v1.0.0 // indirect
 	github.com/imdario/mergo v0.3.6 // indirect
+	github.com/itchyny/json2yaml v0.1.4 // indirect
```

Comment:
This should all be reset - there should be changes to the go.mod here or the go.sum I don't think.